README.md

```diff
@@ -16,7 +16,13 @@ the square brackets (for multiple-choice questions).
 What are constructor functions and the `new` keyword? What is a prototype? Describe an example of when we would use a constructor function versus a prototype.
 
 Your Answer:
-```text
+```
+A constructor is any function that is used as a constructor. It is used with a 'new' keyword. An example of this would be:
+//var Student = function Student(){
+    //...
+//}
+//var student = new Student();
+When we call new Student(), it'll create a new object and sets the constructor property of the object to Student.
 
```

Comment:
Close, be sure to mention the concept of prototypes in relation to constructors.
i.e.
```
A constructor function runs as soon as the function is called, with or without `new`. The `new` keyword calls the function and returns a new object that has the methods and properties of the function's `prototype`.

A prototype is an object, the properties and methods of which other objects inherit.

You would use a constructor when you want a piece of code to run every time you create a new instance of a particular kind of object.
```